featuretools/entityset/entity.py

```diff
@@ -406,9 +406,18 @@ def delete_variable(self, variable_id):
         Remove variable from entity's dataframe and from
         self.variables
         """
-        self.df.drop(variable_id, axis=1, inplace=True)
-        v = self._get_variable(variable_id)
-        self.variables.remove(v)
+        self.delete_variables([variable_id])
```

Comment:
Can you add a test case for this method

Reply:
👍 And do you think we need to keep `delete_variable`?

Reply:
Let's just remove it. It's unused and `delete_variables([variable_id])` achieves the same result